opencensus/metrics/export/gauge.py

```diff
@@ -169,6 +169,64 @@ def to_point_value(self):
         return value_module.ValueDouble(self.value)
 
 
+class CumulativePointLong(GaugePointLong):
+    """A `GaugePointLong` that cannot decrease."""
+
+    def set(self, val):
+        """Set the current value to `val`
+
+        Return without changing the value if `val` is not greater than the
```

Comment:
"is not greater" -> "less"? (looking at the code, we skip if the `val` equals to the value)